### PR TITLE
Allow to configure the read timeout

### PR DESCRIPTION
**Pull request by mrodrigues**
Hi!
I've added the option to configure the `Net::HTTP::Persistent#read_timeout`, and altered the name of the existing configuration for the `Net::HTTP::Persistent#open_timeout` so it's clearer now.

### CODE REVIEW

**Comment by gottfrois** [comment]
Thanks for the PR, however you cannot change an old configuration attribute like that :) We need to keep the compatibility so that it won't break people's code when upgrading.

You should probably add an alias instead.

**Comment by mrodrigues** [comment]
0a59d23784be445eb169edc8987eb4d32b635646
There you go :)

File: spec/processor_spec.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'pry'
```
Comment (gottfrois) [inline spec/processor_spec.rb]: please remove :)

Comment (mrodrigues) [inline spec/processor_spec.rb]: Oops, fixed

**Pull request merged**